.github/workflows/_new_release_notification.yml

```diff
@@ -0,0 +1,24 @@
+name: "New Release Notification"
+
+on:
+  workflow_call:
+    inputs:
+      message:
+        type: string
+        required: true
+
+jobs:
+  discord:
+    runs-on: ubuntu-latest
+    env:
+      DISCORD_WEBHOOK: ${{ secrets.NEW_RELEASE_DISCORD_WEBHOOK }}
+    steps:
+      - name: Notify Discord
+        run: |
+          export TESTED_WITH_DAGGER_VERSION=0.9.10
+
+          curl --fail --silent --show-error https://dl.dagger.io/dagger/install.sh \
+          | DAGGER_VERSION=$TESTED_WITH_DAGGER_VERSION sh
+
+          ./bin/dagger call --mod github.com/gerhard/daggerverse/notify@2024-02-13 \
+            discord --webhook-url env:DISCORD_WEBHOOK --message '${{ inputs.message }}'
```

Comment:
We should pin to a commit SHA here, to ensure that the repo can't be updated to include malicious code.

Not sure if there's a best practice here yet, but I've seen this used in github actions usage as a way of improving visibility + security.

Reply:
All GitHub Actions that we consume today wqe pin to a branch / tag, i.e. `actions/checkout@v3`

Would prefer to keep as is, pinned to this specific tag: `github.com/gerhard/daggerverse/notify@2024-02-13`

Reply:
Not a blocker, but potentially for non official actions, or ones that we develop, it might be worth pinning to specific SHAs. 

Tags aren't immutable, and most releases aren't signed (but that's another alternative).